### ISSUE TITLE
Second trailing closure has incorrect indentation

### ISSUE THREAD

**Issue by Shiaulis** [nicklockwood/SwiftFormat#1100]
Hey! Cannot find a solution here so assuming whether I was looking in a wrong way or it an issue that can be solved.
I'm using swiftformat 0.49.1
My code is:
```
        UIView.animate(withDuration: transitionDuration(using: context)) {
            fromView.transform = CGAffineTransform(translationX: 0, y: fromView.frame.height)
        }
        completion: { finished in
            context.completeTransition(finished)
        }
```
With swiftformat `indent` rule it adds an extra spacing for the last trailing closure:
```
        UIView.animate(withDuration: transitionDuration(using: context)) {
            fromView.transform = CGAffineTransform(translationX: 0, y: fromView.frame.height)
        }
            completion: { finished in
                context.completeTransition(finished)
            }
```

Config I'm using consists of rules:
```
--indent 4
--xcodeindentation enabled
```
Should I change my settings somehow to prevent trailing closure to have an extra spacing in case of 2 or more trailing closures?

**Comment by nicklockwood** [comment]
@Shiaulis this looks like a bug. There doesn't seem to be a workaround for now, other than moving the `completion:` label onto the same like as the closing `}`, i.e.

```swift
UIView.animate(withDuration: transitionDuration(using: context)) {
    fromView.transform = CGAffineTransform(translationX: 0, y: fromView.frame.height)
} completion: { finished in
    context.completeTransition(finished)
}
```

**Comment by alexito4** [comment]
I'm also seeing this happen in a couple of places. 

To be honest this "new" non-trailing closure syntax is something I still don't like at all 😂

**Comment by alexito4** [comment]
Ah btw, since I follow a step by step process when updating I can tell you that I _think_ it was introduced on 0.48.13 (.12 is fine for sure).

**Comment by nicklockwood** [comment]
@alexito4 fixed in 0.49.3
